examples/tracer-web/README.md

```diff
@@ -0,0 +1,34 @@
+# Overview
+
+This example shows how to use [@opentelemetry/tracing](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-tracing) to instrument your JavScript code running in the browser.
+The supplied example uses the `ConsoleSpanExporter()` to export spans into the console output.
```

Comment:
this one is minor take it or leave it

```suggestion
This example uses the `ConsoleSpanExporter()` to export spans to the console.
```

Reply:
I would specify that we are talking about the console in the browser and not the command line that you ran the application from.

Reply:
Updated.